migrations/20201116042500-update-ambulances-unique.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.sequelize.query(
+        `DELETE FROM ambulances WHERE "id" NOT IN (SELECT * FROM (SELECT MIN("id") FROM ambulances GROUP BY "vehicleNumber") x)`,
+        { transaction: t }
+      );
+      queryInterface.addConstraint(
+        'ambulances',
+        {
+          fields: ['vehicleNumber'],
+          type: 'unique',
+        },
+        { transaction: t }
+      );
+    });
+  },
+
+  down: (queryInterface) => {
+    return queryInterface.removeConstraint(
+      'ambulances',
+      'ambulances_vehicleNumber_uk'
```

Comment:
Where do we define this constraint name?

Reply:
Sequelize automatically creates a name for it. Would it make more sense to manually name it for clarity and code readability purposes?

https://sequelize.org/master/class/lib/dialects/abstract/query-interface.js~QueryInterface.html#instance-method-addConstraint
- In the `options.name` row

Reply:
I don't think it really matters, I was just curious.